CHANGELOG.md

```diff
@@ -614,6 +614,9 @@ now encoded as `"[]"` to comply with standard.
   [#9611](https://github.com/Kong/kong/pull/9611)
 - **HTTP-Log**: Support `http_endpoint` field to be referenceable
   [#9714](https://github.com/Kong/kong/pull/9714)
+- **rate-limiting**: Add a new configuration `sync_rate` to the `redis` policy,
+  which synchronizes metrics to redis periodically instead of on every request.
```

Comment:
Why is this specific to redis? Why should it not be possible for cluster strategy as well?

Reply:
It is possible for other strategies. But cluster works well (in performance), and this PR targets to solve the performance issue. We may also support `sync_rate` for `cluster` later.